### PR TITLE
Refactoring errors to not block first import

### PR DESCRIPTION
**Pull request by kosssi**

### CODE REVIEW

File: src/app/replicator/change/change_event_handler.coffee [hunk 1]
```diff
@@ -37,47 +41,86 @@ module.exports = class ChangeEventHandler
...
             callback()
+
+
+    _calendarGetError: (cozyEvent, err, callback) ->
+        msg = 'This calendar have an error to get or create it'
+        @_error cozyEvent, err, msg, callback
+
+
+    _calendarNameError: (cozyEvent, callback) ->
+        msg = 'This event have not calendar name'
```
Comment (nono) [inline src/app/replicator/change/change_event_handler.coffee]: s/not/no/

File: src/app/replicator/change/change_event_handler.coffee [hunk 2]
```diff
@@ -37,47 +41,86 @@ module.exports = class ChangeEventHandler
...
+        @_error cozyEvent, null, msg, callback
+
+
+    _calendarDeleteError: (cozyEvent, err, callback) ->
+        msg = 'This calendar have an error to delete it'
```
Comment (nono) [inline src/app/replicator/change/change_event_handler.coffee]: This calendar can't be deleted due to an error

File: src/app/replicator/change/change_event_handler.coffee [hunk 3]
```diff
@@ -37,47 +41,86 @@ module.exports = class ChangeEventHandler
...
-    _delete: (cozyEvent, androidEvent, callback) ->
-        log.debug "_delete"
 
-        @calendarSync.deleteEvent androidEvent, \
-                AndroidAccount.ACCOUNT, (err, deletedCount) =>
-            log.error err if err
+    _delete: (cozyEvent, androidEvent, callback) ->
+        @calendarSync.deleteEvent androidEvent, @account, (err, deletedCount) =>
+            if err
+                msg = 'This event have an error to delete it'
```
Comment (nono) [inline src/app/replicator/change/change_event_handler.coffee]: This event can't be deleted due to an error

File: src/app/replicator/change/change_event_handler.coffee [hunk 4]
```diff
@@ -37,47 +41,86 @@ module.exports = class ChangeEventHandler
...
+            androidEvent = @cozyToAndroidEvent.transform \
+                    cozyEvent, calendar, @timezone
 
-            androidEvent = @cozyToAndroidEvent.transform cozyEvent, calendar, \
-                @timezone
-            @calendarSync.addEvent androidEvent, \
-                    AndroidAccount.ACCOUNT, callback
+            @calendarSync.addEvent androidEvent, @account, (err) =>
+                if err
+                    msg = 'This event have an error to create it'
```
Comment (nono) [inline src/app/replicator/change/change_event_handler.coffee]: This event can't be created due to an error

**Comment by kosssi** [comment]
Thanks @nono and sorry to my bad english

**Pull request merged**